README.md

```diff
@@ -1,3 +1,190 @@
-# Codex Protocol | Codex Registry API
+# Codex Protocol | Codex Registry API _(service.codex-registry-api)_
 
-Documentation is coming soon! Feel free to [reach out on Telegram](https://t.me/codexprotocol) in the meantime.
+[![standard-readme compliant](https://img.shields.io/badge/readme%20style-standard-brightgreen.svg?style=flat-square)](https://github.com/RichardLitt/standard-readme)
+
+> Node API for Codex Registry
+
+This is the Node API that serves as the off-chain backend for the [Codex Viewer](https://github.com/codex-protocol/web.codex-viewer). Among various other tasks, this API handles user authentication via [JSON Web Tokens](https://www.npmjs.com/package/jsonwebtoken), processes blockchain events via [EEL](https://github.com/codex-protocol/service.eel) & [Agenda](https://www.npmjs.com/package/agenda), manages MongoDB interactions via [Mongoose](https://www.npmjs.com/package/mongoose), and manages socket connections from the front end via [socket.io](https://www.npmjs.com/package/socket.io) & [Redis](https://www.npmjs.com/package/redis).
+
+## Table of Contents
+
+- [Security](#security)
+- [Background](#background)
+- [Install](#install)
+- [Usage](#usage)
+- [API](#api)
+- [Maintainers](#maintainers)
+- [Contribute](#contribute)
+- [License](#license)
+
+
+## Security
+
+It should be noted that this project requires the use of some "secret" variables that should be tailored to your development environment and never committed to the repository. These variables are read by [dotenv](https://www.npmjs.com/package/dotenv) and should be defined in a file called `.env` in the project root.
+
+After cloning the repository, you should duplicate and rename `.env.example` to `.env`, then tailor the variables to your local environment accordingly:
+
+```bash
+$ cp .env.example .env
+$ vi .env
+```
+
+`.env.example` has suitable local development defaults for most variables, so you should only have to set any missing values (i.e. `JWT_SECRET` & `SIGNER_PRIVATE_KEY`.) Descriptions for each variable are listed in [.env.example](.env.example).
+
+## Install
+
+There's a lot involved in setting up this project, so bear with me.
+
+### Clone & Set Up Required Repositories
+
+1. First, clone this repository and all dependency repositories (see below for more details):
+
+```bash
+$ git clone https://github.com/codex-protocol/service.codex-registry-api
+$ git clone https://github.com/codex-protocol/contract.codex-registry
+$ git clone https://github.com/codex-protocol/npm.ethereum-service
+$ git clone https://github.com/codex-protocol/service.eel
+```
+
```

Comment:
I think it's enough to list the dependencies with a link to each one. These will have their own README's with installation instructions.